include/deal.II/base/tensor.h

```diff
@@ -932,6 +961,8 @@ Tensor<0, dim, Number>::operator*=(const OtherNumber &s)
   value *= s;
   return *this;
 }
+#endif
```

Comment:
Actually, I'd rather just always use the same implementation of the function. Isn't that possible?

Reply:
Yes, I can just add the `std::complex` overload in case the device path is compiled.